internal/view/content.html

```diff
@@ -48,15 +50,26 @@
 			el: '#content-scene',
 			mixins: [basePage],
 			data: {
-				modified: "$$.Book.Modified$$"
+				created: "$$.Book.Created$$"
 			},
 			methods: {
-				localtime() {
-					var strTime = this.modified.replace(" ", "T");
-					if (!strTime.endsWith("Z")) {
-						strTime += "Z";
+				createdTime() {
```

Comment:
I see that we are only using `dayjs` here. Maybe we can remove this dependency and provide a vanilla JS parseable datetime? If we return ISO strings (and we should) we can do `new Date(isoString)` and that should be a parsed datetime, we can then display that however we like. What do you think?